src/Commands/IntakeArmOn.cpp

```diff
@@ -9,13 +9,14 @@
 // it from being updated in the future.
 
 
-#include "IntakeDown.h"
+#include "IntakeArmOn.h"
 
 // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTOR
 
-IntakeDown::IntakeDown(): Command() {
+IntakeArmOn::IntakeArmOn(double mspeed): Command() {
```

Comment:
rename `mspeed` to `speed_`. 

The `m` prefix is traditionally used to represent a "member" variable.